.github/CODEOWNERS

```diff
@@ -3,13 +3,13 @@
 
 * @mdragon # project lead
 
-/documentation/  @andycochran @btabaska @chouinar @doug-s-nava @joshtonava @mdragon @myduong-navapbc @widal001 @prasnava @jakobpederson @ErinPattisonNava @glenblosser-nava # everyone
+/documentation/  @andycochran @btabaska @chouinar @doug-s-nava @joshtonava @mdragon @myduong-navapbc @widal001 @prasnava @jakobpederson @ErinPattisonNava @glenblosser-nava @hao10282025-sudo # everyone
```

Comment:
(optional) maybe through these changes into their own PR, they seem pretty unrelated to this issue.